lib/upsert/connection/Java_ComMysqlJdbc_JDBC4Connection.rb

```diff
@@ -6,9 +6,12 @@ class Connection
     class Java_ComMysqlJdbc_JDBC4Connection < Connection
       include Jdbc
 
-      # ? backtick?
       def quote_ident(k)
-        DOUBLE_QUOTE + k.to_s.gsub(DOUBLE_QUOTE, '""') + DOUBLE_QUOTE
+        if metal.useAnsiQuotedIdentifiers
+          DOUBLE_QUOTE + k.to_s.gsub(DOUBLE_QUOTE, '""') + DOUBLE_QUOTE
+        else
+          BACKTICK + k + BACKTICK
```

Comment:
Apparently you're allowed to (https://github.com/tgriesser/knex/issues/753).  Looks like you escape it by doubling-up the backtick.  I'll add that.